Realm/RLMUtil.hpp

```diff
@@ -27,6 +27,9 @@
 
 @class RLMProperty;
 
+NSException *RLMException(NSString *message, NSDictionary *userInfo = nil);
+NSException *RLMException(std::exception exception);
```

Comment:
`std::exception const& exception`

Passing polymorphic types by value doesn't work (you get a `std::exception` copy inside the function rather than a copy with the original type).